```
boost/test/tools/detail/print_helper.hpp:53:13: error: static assertion failed:
    Type has to implement operator<< to be printable
    BOOST_STATIC_ASSERT_MSG( (boost::has_left_shift<std::ostream,T>::value), ... )
  • 'boost::integral_constant<bool, false>::value' evaluates to false
...:55:18: error: no match for 'operator<<' (operand types are 'std::ostream'
    and 'const std::vector<seastar::basic_sstring<char, unsigned int, 15> >')
```

I'm building with -DSeastar_DEPRECATED_OSTREAM_FORMATTERS=OFF (to drop the deprecated).

```
#ifdef SEASTAR_DEPRECATED_OSTREAM_FORMATTERS
std::ostream& operator<<(std::ostream& os, const std::vector<T>& v) {
#endif
```

--- a/apps/memcached/tests/test_ascii_parser.cc
+++ b/apps/memcached/tests/test_ascii_parser.cc
@@ -27,6 +27,8 @@
 #include "ascii.hh"
 #include <seastar/core/loop.hh>
 
+BOOST_TEST_DONT_PRINT_LOG_VALUE(std::vector<seastar::sstring>)
+
 using namespace seastar;
 using namespace net;
 using namespace memcache;

--- a/tests/unit/simple_stream_test.cc
+++ b/tests/unit/simple_stream_test.cc
@@ -24,6 +24,8 @@
 #include <boost/test/unit_test.hpp>
 #include <seastar/core/simple-stream.hh>
 
+BOOST_TEST_DONT_PRINT_LOG_VALUE(std::vector<char>)
+
 using namespace seastar;
 
 template<typename Input, typename Output>

--- a/tests/unit/websocket_test.cc
+++ b/tests/unit/websocket_test.cc
@@ -12,6 +12,8 @@
 #include <seastar/util/memory-data-source.hh>
 #include "loopback_socket.hh"
 
+BOOST_TEST_DONT_PRINT_LOG_VALUE(std::vector<seastar::sstring>)
+
 using namespace seastar;
 using namespace seastar::experimental;
 using namespace std::literals::string_view_literals;

